.coveragerc

```diff
@@ -12,9 +12,6 @@ source =
 [report]
 # small library, don't fail when running without C-extension
 fail_under = 50.00
-skip_covered = true
-skip_empty = true
-show_missing = true
```

Comment:
This only influences how the report is displayed, but not the actual coverage metric FYI.